frontend/app/routes/protected/multi-channel/layout.tsx

```diff
@@ -0,0 +1,10 @@
+import { Outlet } from 'react-router';
+
+import type { Route } from './+types/layout';
+
+import { useTabId } from '~/hooks/use-tab-id';
+
+export default function Layout({ actionData, loaderData, matches, params }: Route.ComponentProps) {
+  const tabId = useTabId(); // ensure we always have a tabId generated
+  return <Outlet context={{ tabId }} />;
+}
```

Comment:
This page probably doesn't need a layout.tsx file.

The reason that the in-person case needs this is because we have to store data in session while the user navigates from step to step, while also being able to handle multiple tabs without clobbering one another.

If the multi-channel flow is just a single page which doesn't store anything in session, this almost certainly is not needed.